html-folder/alerts.html

```diff
@@ -114,28 +116,20 @@ <h1 class="text-shadow"> PISTON <span class="highlight-orange-text">UI</span></h
     <div class="universal-container">
         <aside class="aside-nav box-shadow-right">
             <div class="aside-container-pri ">
-                <a href="" class="link-btn">
+                <a href="../index.html" class="link-btn">
                     <h2>Get Started</h2>
                 </a>
                 <ul class="aside-ul">
                     <li class="aside-ul-li margin-bot-med">
                         <a href="../index.html" class="aside-link link-btn ">Introduction</a>
```

Comment:
remove space in class name

Reply:
Thanks for the review and your value able suggestions I will make the changes suggested by you